src/azure-account.ts

```diff
@@ -773,17 +785,71 @@ async function getEnvironments(): Promise<Environment[]> {
 
 	const config = workspace.getConfiguration('azure');
 	const ppe = config.get<Environment>('ppe');
+	// get api profile from user setting, this needs to be true for running azure stack
+	const apiProfile = config.get<boolean>('target_azurestack_api_profile');
```

Comment:
I think this should be moved into the `getPpeEnvironments` function since it is only needed there